volatility3/framework/plugins/linux/pagecache.py

```diff
@@ -420,8 +420,54 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
         ]
 
     @staticmethod
+    def read_inode_content(
+        inode: inodeClass,
+        buffer: IO,
```

Comment:
You might want to choose a more appropriate parameter name, as the only user so far of this function isn’t actually a buffer. Also the method name doesn't look right since it reads and writes the content.